src/server/json_family_test.cc

```diff
@@ -262,4 +262,214 @@ TEST_F(JsonFamilyTest, Toggle) {
   EXPECT_EQ(resp, R"([true,false,1,null,"foo",[],{}])");
 }
 
+TEST_F(JsonFamilyTest, NumIncrBy) {
+  string json = R"(
+    {"e":1.5,"a":1}
+  )";
+
+  auto resp = Run({"set", "json", json});
+  ASSERT_THAT(resp, "OK");
+
+  resp = Run({"JSON.NUMINCRBY", "json", "$.a", "1.1"});
+  ASSERT_THAT(resp, ArgType(RespExpr::ARRAY));
```

Comment:
ok, this is not what's expected from the spec and I also missed this when reviewing your previous PRs.

```
json.set foo $ '{"a": 1 }'
json.NUMINCRBY foo $.a 1.1
"[2.1]"
```
see also here: https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/json-numincrby.html

in other words, DF should always return a string. Always a single string and not an array. The string is a representation of a json array in this case but you never return a RESP array. so all response code will have to change for all commands IMHO 🤷🏼